images/gcb-docker-gcloud/Dockerfile

```diff
@@ -32,6 +32,10 @@ WORKDIR /workspace
 RUN echo http://dl-cdn.alpinelinux.org/alpine/latest-stable/community >> /etc/apk/repositories && \
     apk --no-cache add curl python3 py-crcmod bash libc6-compat openssh-client git gnupg docker-cli make
 
+# Install jq
+RUN curl -o /usr/bin/jq -fsSLO https://github.com/jqlang/jq/releases/download/jq-1.8.0/jq-linux-amd64
```

Comment:
If we *do* do this, we should at the very least be doing an integrity check (we should be for the others already here but that's not related to this PR), if not pinning the hash outright